src/rustdoc/membership.md

```diff
@@ -0,0 +1,116 @@
+# Membership
+This section discusses membership in the rustdoc team.
+
+## The path to membership
+
+People who are looking to contribute on the rustdoc tool generally start on either fixing bugs
+or implementing a new feature. If you need guidance or help, don't hesitate to ask on the
+[t-rustdoc channel on Zulip](https://rust-lang.zulipchat.com/#narrow/channel/266220-t-rustdoc)!
+
+## Rustdoc member
+Once an individual has been contributing regularly for some time, they can be promoted to the
+level of a **rustdoc team member** (see the section on [how decisions are made][hdam] below).
+This title indicates that they are someone who contributes regularly.
+
+It is hard to define the precise conditions when such a promotion is appropriate. Being promoted
+to member is not just a function of checking various boxes. But the general sense is that someone
+is ready when they have demonstrated three things:
+
+- "Staying power" -- the person should be contributing on a regular basis in some way. This might
+  for example mean that they have completed a few projects.
+- "Independence and familiarity" -- they should be acting somewhat independently when taking on
+  tasks, at least within the scope of their "rustdoc area". They should plausibly be able to mentor
+  others on simple PRs.
+- "Cordiality" -- rustdoc team members will be part of the Rust organization and are held to a
+  higher standard with respect to the [Code of Conduct][CoC]. They should not only obey the
+  letter of the CoC but also its spirit.
+
+[CoC]: https://www.rust-lang.org/policies/code-of-conduct
+
+Being promoted to member implies a number of privileges:
+
+- Members have `r+` (approve a pull request) privileges and can do reviews (they are expected to
+  use those powers appropriately, as discussed previously). They also have access to control
+  perf/rustc-timer and other similar bots. See the documentation for `bors` and `r+`
+  [here](https://rustc-dev-guide.rust-lang.org/contributing.html#r-1).
+
+  Tip: some baseline rules around bors permissions are: don't do a `try` build unless you have
+  done a check for malicious code first and don't `r+` unless you are reasonably confident that
+  you can effectively review the code in question.
+- Rustdoc team members are members of the Rust organization so they can modify labels and be
+  assigned to issues.
+- Members become a part of the `rust-lang/rustdoc` team on GitHub, so that they receive pings
+  when people are looking to address the team as a whole.
+- Members are [listed] on the rust-lang.org web page.
+
+It also implies some obligations (in some cases, optional obligations):
+
+- Members may take part in various other [maintainer activities] to help the team.
+- Members are held to a higher standard than ordinary folk when it comes to the [Code of
+  Conduct][CoC].
+
+[listed]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
```

Comment:
```suggestion
- Members are listed on [the rust-lang.org web page].

It also implies some obligations (in some cases, optional obligations):

- Members may take part in various other [maintainer activities] to help the team.
- Members are held to a higher standard than ordinary folk when it comes to the [Code of
  Conduct][CoC].

[the rust-lang.org web page]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
```

putting the link here will make more sense for anyone who is just looking at link names, either because they're skimming or if they're using the "jump to next link" feature of a screen reader.